rocksdb_admin/admin_handler.cpp

```diff
@@ -1045,14 +1053,17 @@ void AdminHandler::async_tm_startMessageIngestion(
     kafka_watcher_map_[db_name] = kafka_watcher;
   }
 
+  int64_t message_count = 0;
+  const int message_interval = 1000; // use glfag or zk to configure this
```

Comment:
can we add a gflag in this file and we don't need to capture message_interval in the lambda below.

Reply:
Adding it as a gflag for now. In the future, we might end up needing different intervals for each use case, so I'm leaning towards having that as part of zk config.